frontend/src/components/studies/StudyEditForm.tsx

```diff
@@ -114,6 +122,16 @@ const StudyEditForm = ({ workflows, isCreatingStudy, onSubmit, formErrors }: Cre
 					onChange={workflowWasSelected}
 				/>
 			</Item>
+			<Item
+				name="description"
+				label="Description"
+				{...itemValidation('description')}
+			>
+				<Input
```

Comment:
The content may overflow from the single line input field. I would maybe try the textarea with autosize prop instead. This leave the component small if not much text is input, but let the user see the whole content if more than one line of input is required. Might also be nice when entering the description that the "enter" not be captured by the submit button but instead insert a new line in the field (optional) ...